gradle-diff-task-dependencies/plugin/src/main/groovy/com/novoda/gradle/diffdependencytask/DiffDependencyTaskPlugin.groovy

```diff
@@ -0,0 +1,46 @@
+package com.novoda.gradle.diffdependencytask
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+import java.util.regex.Pattern
+
+class DiffDependencyTaskPlugin implements Plugin<Project> {
+
+    private static String TASK_CONFIG_PROPERTY_NAME = 'diffDependency'
+
+    @Override
+    void apply(Project project) {
+
+        project.extensions.add(TASK_CONFIG_PROPERTY_NAME, new DiffDependencyTaskConfig())
```

Comment:
It can be used by clients, see https://github.com/novoda/spikes/pull/183/files#diff-fb0c667cda520bf910e0ca1878413fd9R39